tests/e2e.sh

```diff
@@ -100,3 +99,48 @@ docker run -u "$(id -u)" -v "$(pwd)/${CHARTS_SRC}":/source gcr.io/kpt-functions/
 assert_dir_exists default
 assert_contains_string default/secret_my-mongodb.yaml "my-mongodb"
 assert_contains_string default/secret_my-redis.yaml "my-redis"
+
+testcase "docker_istioctl_analyze_success"
+kpt pkg get https://github.com/istio/istio.git/samples/addons .
+cat >fc.yaml <<EOF
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: my-config
+  annotations:
+    config.k8s.io/function: |
+      container:
+        image:  gcr.io/kpt-functions/istioctl-analyze:dev
+    config.kubernetes.io/local-config: 'true'
+data:
+  "flags": [ "--recursive" ]
+  "--use-kube": "false"
+EOF
+docker run -i -u "$(id -u)" --mount type=bind,src="$(pwd)",dst=/source gcr.io/kpt-functions/read-yaml:dev -i /dev/null -d source_dir=/source/addons |
```

Comment:
I've mentioned this previously: Let's not add any more `docker run` tests. Just use `kpt fn run`.

Reply:
Done